docs/frontend-architecture.md

```diff
@@ -67,15 +67,6 @@ environment variables to the outside world. You can extend this endpoint in
 using the `useRuntimeData` hook; you can find this in
 `/frontend/src/hooks/api/runtimeData.ts`.
 
-## Add a feature flag
```

Comment:
IIRC Se Yeon is writing docs on how feature flagging with Waffle will be done, so I didn't include replacement docs for this.